README.md

```diff
@@ -357,14 +362,19 @@ Generate shared secret for Shared Secret Protocol (SSP).
 **Returns**: <code>Object</code> - Object containing destination address and shared secret  
 <a name="module_Receiver..createReceiver..listen"></a>
 
-#### createReceiver~listen() ⇒ <code>Promise.&lt;null&gt;</code>
+#### createReceiver~listen(callback) ⇒ <code>Promise.&lt;null&gt;</code>
 Listen for incoming transfers and automatically fulfill
 conditions for transfers corresponding to requests this
 receiver created.
 
 **Kind**: inner method of <code>[createReceiver](#module_Receiver..createReceiver)</code>  
 **Returns**: <code>Promise.&lt;null&gt;</code> - Resolves when the receiver is connected  
 **Emits**: <code>[incoming](#event_incoming)</code>, <code>incoming:&lt;requestid&gt;</code>, <code>incoming:ssp:&lt;token&gt;</code>  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| callback | <code>function</code> | (optional) called when an incoming payment is about to be fulfilled. throwing an error will stop from fulfilling. |
```

Comment:
Not a fan of depending on a callback throwing an error. That's unusual. Either it should take a function that gets the request and a callback or it should take a function that returns a promise.